### PR TITLE
[native] Modify README.md in example_face_detection_tflite

### PR DESCRIPTION
**Pull request by soo-ni**
Modify README.md in example face detection file for git PR test

### CODE REVIEW

File: native/example_face_detection_tflite/README.md [hunk 1]
```diff
@@ -3,4 +3,5 @@
 
 This example passes camera video stream to a neural network using **tensor_filter**. The neural network predicts faces of people in input stream
 
-To be updated..
+To be updated...
```
Comment (gichan-jang) [inline native/example_face_detection_tflite/README.md:6]: Modify it again as practice.
```suggestion
To be updated.
```

**Review by dreamfinder2019 — APPROVED** [review]
"To be updated." is more appropriate than "To be updated..". I'll approve this.

**Review by YongjoonSeo — APPROVED** [review]
LGTM.

File: native/example_face_detection_tflite/README.md [hunk 2]
```diff
@@ -3,4 +3,5 @@
 
 This example passes camera video stream to a neural network using **tensor_filter**. The neural network predicts faces of people in input stream
 
-To be updated..
+To be updated.
```
Comment (wooksong) [inline native/example_face_detection_tflite/README.md:6]: If you want to modify this, write a sentence instead of a phrase.

**Review by gichan-jang — APPROVED** [review]

**Pull request merged**